resources/puppetlabs/lein-ezbake/template/global/ext/cli/start.erb

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
I'd recommend either adding `set -e` here or adding `-e` to the bash invocation. `-e` means any non-zero exit will halt the script, which can be good, for example if on line 13 we don't have permissions to create a new directory.